docs/UserGuide.md

```diff
@@ -125,12 +131,12 @@ Examples:
 
 ### Adding favourites : `fav`
 
-Adds the contact to your list of favourite contacts and view them.
+Toggles the favourite status of your contacts. 
+Favourited contacts **can be un-favourited** by running the same command on the contact again.
 
 Format:
 
 - `fav INDEX`
```

Comment:
Might want to keep this to 1 line, e.g. `Format: fav INDEX` to be consistent with the other features. Just a suggestion!